src/pages/home/home--logged-in.stories.js

```diff
@@ -664,46 +664,46 @@ const Template = (args) => (
           variant="positive"
         >
-          <li class="link-list__item" slot="link-list__item">
-            <sbb-link
-              href-value="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
-              text="Rückerstattungen"
-              text-size="xs"
-              variant="positive"
-            ></sbb-link>
-          </li>
-          <li class="link-list__item" slot="link-list__item">
-            <sbb-link
-              href-value="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
-              text="Fundbüro"
-              text-size="xs"
-              variant="positive"
-            ></sbb-link>
-          </li>
-          <li class="link-list__item" slot="link-list__item">
-            <sbb-link
-              href-value="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
-              text="Beschwerden"
-              text-size="xs"
-              variant="positive"
-            ></sbb-link>
-          </li>
-          <li class="link-list__item" slot="link-list__item">
-            <sbb-link
-              href-value="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
-              text="Lob aussprechen"
-              text-size="xs"
-              variant="positive"
-            ></sbb-link>
-          </li>
-          <li class="link-list__item" slot="link-list__item">
-            <sbb-link
-              href-value="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
-              text="Sachbeschädigung melden"
-              text-size="xs"
-              variant="positive"
-            ></sbb-link>
-          </li>
+          <sbb-link
+            href-value="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
+            text="Rückerstattungen"
+            text-size="xs"
+            variant="positive"
+          >
+            Rückerstattungen
+          </sbb-link>
+          <sbb-link
+            href-value="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
+            text="Fundbüro"
+            text-size="xs"
+            variant="positive"
+          >
+            Fundbüro
+          </sbb-link>
+          <sbb-link
+            href-value="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
+            text="Beschwerden"
+            text-size="xs"
+            variant="positive"
+          >
+            Beschwerden
+          </sbb-link>
+          <sbb-link
+            href-value="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
+            text="Lob aussprechen"
+            text-size="xs"
+            variant="positive"
```

Comment:
All links in the story have an old api. text is now the default slot, variant="positive" is not existing (default), href-value should be "href" only.